.github/workflows/ci.yml

```diff
@@ -10,25 +10,22 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 16.x]
+        node-version: [12, 17]
```

Comment:
Since 16 is LTS we should test it?

Reply:
If something works on 12 and 17, I thought it should work on 14 as well like we do in our other libraries? But I can bring it back.

Reply:
We should test all LTS versions as they can differ and have breaking changes (although very unlikely that they will affect helix tbh).